src/plugins/PluginAPI.cpp

```diff
@@ -301,7 +301,7 @@ APICALL std::vector<SFunctionMatch> HyprlandAPI::findFunctionsByName(HANDLE hand
             count++;
         }
 
-        return SYMBOLSDEMANGLED.substr(pos, SYMBOLSDEMANGLED.find('\n', pos + 1) - pos);
+        return SYMBOLSDEMANGLED.substr(pos + 1, SYMBOLSDEMANGLED.find('\n', pos + 1) - pos - 1);
```

Comment:
if pos is 0, we can't add 1

Reply:
Fixed in 2c9bf9022f1773006e8fe6768d34ad8308483233